### PR TITLE
Update javadocs, code quality and authorship

### PR DESCRIPTION
**Pull request by AndreWongZH**

### CODE REVIEW

File: src/main/java/seedu/duke/unused/Event.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public String getStatus() {
+        return (isDone ? DONE_STATUS : NOT_DONE_STATUS);
+    }
+
+    @Override
+    public int compareTo(Event o) {
```
Comment (durianpancakes) [inline src/main/java/seedu/duke/unused/Event.java:18]: Is our superclass containing the start and end time still present?

**Review by durianpancakes — APPROVED** [review]
Everything looks good to me. I will be working to ensure that I match your standards as well.

File: src/main/java/seedu/duke/unused/Event.java (hunk 1 above)
Comment (AndreWongZH) [reply src/main/java/seedu/duke/unused/Event.java:18]: This is for the unused codes, which is the done/notdone feature

Comment (durianpancakes) [reply src/main/java/seedu/duke/unused/Event.java:18]: Understood. Thank you!

**Pull request merged**